src/style/sections/libraries/libraries-section.less

```diff
@@ -21,86 +21,268 @@
  *
  */
  
- @graphic-drag-preview-size: 64px;
+@graphic-drag-preview-size: 64px;
```

Comment:
This is just a question, but generally pixel-based size specs are bad in Design Space because are UI is of variable size. But, I know that we're somewhat constrained by the thumbnails, etc. Is it not possible to use rems more thoroughly here?